src/domains/sign/io_sign_support_func.tf

```diff
@@ -0,0 +1,186 @@
+locals {
+  io_sign_support_func = {
+    app_settings = {
+      FUNCTIONS_WORKER_RUNTIME                        = "node"
+      WEBSITE_VNET_ROUTE_ALL                          = "1"
+      WEBSITE_ADD_SITENAME_BINDINGS_IN_APPHOST_CONFIG = "1"
+      WEBSITE_DNS_SERVER                              = "168.63.129.16"
+      WEBSITE_RUN_FROM_PACKAGE                        = "1"
+      FUNCTIONS_WORKER_PROCESS_COUNT                  = 4
+      AzureWebJobsDisableHomepage                     = "true"
+      NODE_ENV                                        = "production"
+      CosmosDbConnectionString                        = module.cosmosdb_account.connection_strings[0]
+      CosmosDbIssuerDatabaseName                      = module.cosmosdb_sql_database_issuer.name
+      CosmosDbUserDatabaseName                        = module.cosmosdb_sql_database_user.name
+      PdvTokenizerApiBasePath                         = "https://api.uat.tokenizer.pdv.pagopa.it"
+      PdvTokenizerApiKey                              = module.key_vault_secrets.values["TokenizerApiSubscriptionKey"].value
+    }
+  }
+}
+
+module "io_sign_support_func" {
+  source = "git::https://github.com/pagopa/terraform-azurerm-v3.git//function_app?ref=v4.1.6"
```

Comment:
Since this is a new function, I would use the new version of the module [`v6.0.1`](https://github.com/pagopa/terraform-azurerm-v3/tree/v6.0.1/function_app) which creates [`azurerm_linux_function_app`](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/linux_function_app) instead of the deprecated [`azurerm_function_app`](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/function_app).

A migration guide is available in our internal wiki at <https://pagopa.atlassian.net/wiki/search?text=Azure%20-%20Function%20App>

In your case, since it is a new function there's no warn of possible issue due to migration.